### DISCUSSION TITLE
Add FybrikApplication ID to the connectors calls

### DISCUSSION THREAD

**Discussion by roytman** [fybrik/fybrik#1794] [Ideas]
All connectors calls are done in a context of a FybrikApplication. 
I suggest adding to the requests the application ID, so we will be able to connect connectors logs with a relevant FybrikApplication.
 
@simanadler @shlomitk1 thought ?

**Comment by shlomitk1** [comment]
Ok. I would suggest creating a base struct that contains appId and encapsulating it by all Request structures.
If changes to API are made, we are also missing a message string in the Response structures (there was a request to add it and propagate to the app status). In general, there are some inconsistencies in the responses of different operations.
